wordpress-mobile/gutenberg-mobile
# [RNMobile] Add support Reusable blocks

`Gutenberg PR` -> https://github.com/WordPress/gutenberg/pull/25265

**Second part (adding Reusable blocks to the inserter menu):**
`Gutenberg PR` -> https://github.com/WordPress/gutenberg/pull/25383
`Gutenberg-mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/2644

Related to https://github.com/wordpress-mobile/gutenberg-mobile/issues/2575

This PR is the first part of the issue Add Support to Insert Existing Reusable Blocks from the Editor.

The scope of this issue is limited to add Reusable blocks to the inserter menu, however due to the need of registering the Reusable block type (core/block), I also had to add support for presenting it in the editor.

The edition support will be added in this issue, so for now this PR only provides the preview mode of this type of block.

To test:
Check test steps in Gutenberg PR https://github.com/WordPress/gutenberg/pull/25265

PR submission checklist:
